tools/login/actions/login.php

```diff
@@ -32,15 +39,20 @@
 // Lecture des parametres de l'action
 
 // NOTE: à mettre dans la classe ?
+
+// urls after successfull login and successfull logout
+
+$loggedinurl = $this->GetParameter('loggedinurl');
+$loggedouturl = $this->GetParameter('loggedouturl');
+
 // url d'inscription
 $signupurl = $this->GetParameter('signupurl');
 // si pas d'url d'inscription renseignée, on utilise ParametresUtilisateur
-if (empty($signupurl) || $signupurl === "0") {
+
+if (empty($signupurl) && (!(isset ($signupurl) && $signupurl === "0"))) {
```

Comment:
```suggestion
if (empty($signupurl)) {
```
devrait suffire et faire la même chose en étant plus facile à lire

Reply:
en fait, je me dis que ce bout de code n'est pas clair et qu'il nous faudrait un `bool` du genre
```php
$noSignupButton = $signupurl === "0";
```

puis plus loin on écrit
```php
if (!$noSignupButton){
  if (empty($signupurl ){
    ...
  } else {
   ..; 
 }
```
ce serait plus facile à debugger, qu'en penses-tu @VeveQNV ?

et on envoie cet variable au template

Reply:
Possiblement pour ta proposition. Néanmoins cela fait des modifications un peu partout : aide, etc...
Pour le test je l'ai réfléchis : empty return true avec la valeur "0"
Ta proposition ne marchera pas.

Reply:
la première proposition ne marcherait pas mais la deuxième oui

Reply:
au temps pour moi. oui c'est une bonne idée.